doto-frontend/src/components/pages/Calendar/Calendar.js

```diff
@@ -53,6 +84,24 @@ const Calendar = () => {
         setOpen(false);
     };
 
+    const handleStatsOpen = () => {
+        if (statsOpen) {
+            setStatsOpen(false);
+        } else {
+            var medTasks = tasks.filter(function(task) {
+                return task.priority === 20 && task.isComplete;
+            });
+            var lowTasks = tasks.filter(function(task) {
+                return task.priority === 30 && task.isComplete;
+            });
+            var highTasks = tasks.filter(function(task) {
+                return task.priority === 10 && task.isComplete;
+            });
```

Comment:
Some small syntax things here too - try not to use `var`. Also, the rest of the app uses arrow functions over the regular `function` keyword functions. These should really be enforced in the style rules IMO.